site/en/docs/webstore/using_webstore_api/index.md

```diff
@@ -121,20 +144,20 @@ https://www.googleapis.com/upload/chromewebstore/v1.1/items
 
 {% Aside %}
 
-**Note**: For a full description of the insert method, see [Items:Insert][6].
+For a full description of the insert method, see [Items:Insert][cws-api-insert].
 
 {% endAside %}
 
 ### Uploading a package to update an existing store item {: #uploadexisitng }
 
 ```text
-Endpoint: https://www.googleapis.com/upload/chromewebstore/v1.1/items/$APP_ID
+Endpoint: https://www.googleapis.com/upload/chromewebstore/v1.1/items/$ITEM_ID
 Type: PUT
 Header Parameters: $TOKEN: the access token
 Body content: the package file to upload
 ```
```

Comment:
I'm not wild about this presentation, but I don't want to bog down this PR with any more stylistic changs than I already have.

Reply:
@jpmedley, do you know if we have an established convention for placeholder values in code snippets? I found the [Placeholders page](https://developers.google.com/style/placeholders#placeholders) in the Google developer documentation style guide, but I don't know if the Markdown guidance applies to dcc or devsite.

Reply:
FYI, I created a feature request for `<var>` support: #/1902

Reply:
We don't have any good formatting options because we can't do anything inside the backticks. We need to handle this with text. After the example:

Where:
* `ITEM_ID` is ...
* `TOKEN` is ...

If you only have one variable to define, don't use bullets. Make it a single sentence. 

I'm not crazy about the '`$`'. Granting that I don't know anything about the conventions of extensions, my first question is do I have to define or is this a variable that's provided to me? With the dollar sign, it looks like a bash variable.

Reply:
This is a placeholder value for the purposes of documentation. In this case we're not showing JavaScript, so it's not something you'd have to define or something provided to you. Instead, the expectation is that you'd use this information to craft your implementation. For example, I might take this data

> ```text
> Endpoint: https://www.googleapis.com/upload/chromewebstore/v1.1/items/$APP_ID
> Endpoint: https://www.googleapis.com/upload/chromewebstore/v1.1/items/$ITEM_ID
> Type: PUT
> Header Parameters: $TOKEN: the access token
> Body content: the package file to upload
> ```

and use it to write this in my application

```js
class CwsApiManager {
  // OAuth token used to communicate with the CWS API. Set during initial
  // construction.
  token;

  constructor(oauthToken) {/* ... */}

  async uploadItem({id, zipBlob: zip}) {
    let url = `https://www.googleapis.com/upload/chromewebstore/v1.1/items/${id}`;
    let options = {
      method: 'PUT',
      headers: {Authentication: `Bearer ${this.token}`},
      body: zip,
    }
    return fetch(url, options);
  }
}
```

Reply:
I _thought_ our existing convention was to use angle brackets (for example `<PLACEHOLDER>`). I did a quick search and didn't see this pattern in practice.

After a bit more digging I think I found a pervasive bug where we used `<` instead of `&lt;` for placeholder values. For example, the MV2 [Architecture overview page](https://developer.chrome.com/docs/extensions/mv2/architecture-overview/) shows `chrome-extension:///` instead of `chrome-extension://<extensionID>/<pathToFile>`. I'm going to create an issue and start working on a fix for this.

Reply:
If the Google standard is brackets, we wouldn't follow it because of the danger of being confused for an HTML tag or a web component.

Reply:
[The Google standard](https://developers.google.com/style/placeholders#placeholders) is to decorate the a placeholder name (all caps with underscores separating words) with var tags (for example, `<var>PLACEHOLDER_NAME</var>`). Failing that, the [Placeholders in code blocks](https://developers.google.com/style/placeholders#placeholders-in-code-blocks) section states:

> * In Markdown on a non-DevSite site, wrap the code block in a code fence (```). Inside a code fence, you can't apply formatting like bold or italic.
> 
>     ````md
>     ```
>     PLACEHOLDER_NAME
>     ```
>     ````

I guess for the moment we can go with that guidance and simply avoid decorating placeholder names with angle brackets or other special syntax.

Reply:
Whatever the Google standard is, you really should be consistent with existing extensions documents whatever that looks like. I should have opened with that. Conforming to the Google standard is something that should be done all at once rather that piecemeal so that a single documentation set can is consistent with itself.

Certain restrictions on language is the only Google writing standard an individual writing team is obligated to. We generally adopt the Google standard because it's more convenient that writing one from scratch.

Reply:
>Instead, the expectation is that you'd use this information to craft your implementation. For example, I might take this data

Then I think you're way overthinking this. The brackets won't solve the basic issue, which I didn't understand until now. It's because I didn't have both pieces in view. 

1. First name the placeholders in a more meaningful way. For example: YOUR_APP_ID, YOUR_TOKEN, or whatever works for your content.
2. Use those placeholders _identically_ in both locations.

As I said, we're under no obligation to follow the Google style guide to the letter. Frankly, the convention they decided on doesn't get around the obligation to make these things clear in the text, which is always the first rule of writing. If you do that, the brackets don't add any new information.

Reply:
Thanks, @jpmedley. That all makes sense to me.

If you don't mind, at this point I'm inclined to try to leave the placeholders as they are for now and fix them up in a separate PR.